package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "web-ai-demos",
+  "version": "1.0.0",
+  "description": "This repository contains demos related to client-side (in-browser) AI.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
```suggestion
```

Reply:
is that accidental?

Reply:
It's intended. This seems superficial as it doesn't do anything.